File: frontend-proj-app/src/app/model/CartCredentialsDTO.ts
export class CartCredentialsDTO{
  idProductsFromCart:any[];
  id:number;
  sum:string;
  iban:string;
  voucher:string;


  constructor(idProductFromCart:any[],id:number,sum:string,iban:string,voucher:string){
    this.idProductsFromCart=idProductFromCart;
    this.id=id;
    this.sum=sum;
    this.iban=iban;
    this.voucher=voucher;
  }


}

File: resurse/src/main/java/com/proiect_PS/resurse/dto/CartCredentialsDTO.java
package com.proiect_PS.resurse.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Positive;
import java.util.List;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class CartCredentialsDTO {
    @NotNull
    private List<String>  idProductsFromCart;
    Long id;
    @Positive
    String sum;
    String iban;
    String voucher;
}
